### ISSUE TITLE
Failed to install stack-pr through pipx.

### ISSUE THREAD

**Issue by rz-robsn** [modularml/stack-pr#23]
I get a hash mismatch error when trying to install this package from pipx : 

```
╭─robson in ~/projects/stack-pr on 0.1.1✔ 24-09-09 - 10:53:24
╰─⠠⠵ pipx install .
  error: subprocess-exited-with-error
  
  × pip subprocess to install build dependencies did not run successfully.
  │ exit code: 1
  ╰─> [7 lines of output]
      Collecting pdm-backend
        Downloading pdm_backend-2.3.3-py3-none-any.whl.metadata (5.1 kB)
      ERROR: THESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.
          pdm-backend from https://files.pythonhosted.org/packages/eb/fe/483cf0918747a32800795f430319ec292f833eb871ba6da3ebed4553a575/pdm_backend-2.3.3-py3-none-any.whl:
              Expected sha256 639679a37dd96f007466e6d205d376b0b9721b148a05dfc282fde771c6bc1dd7
                   Got        83799b9dc2746117d6d1306b8093cb2dac1c50be3a28768518aa50344b94819d
      
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× pip subprocess to install build dependencies did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
Cannot determine package name from spec '/Users/robson/projects/stack-pr'. Check package spec for errors.
```

**Comment by ZolotukhinM** [comment]
@zbowling do you know what might be causing this?

**Comment by zbowling** [comment]
Unfortunately i'm not seeing this locally. When you pipx a local checkout, it pulls build dependencies, including PDM in this case, from pypi. It looks what your machine downloaded didn't validate the checksum. In this case it's not even the actual wheel but the metadata for the wheel. Checked the checksum for PDM backend metadata when I tried and it's valid when I downloaded. 

The only thing I think of is to try again. Checksums could fail for network issues, hosting issue with pypi, or local disk/memory issues.
